linux/switchtec.h

```diff
@@ -135,9 +144,11 @@ struct sys_info_regs {
 	u32 table_format_version;
 	u32 partition_id;
 	u32 cfg_file_fmt_version;
+	u16 bl2_running;
 	u16 cfg_running;
-	u16 img_running;
-	u32 reserved2[57];
+	u16 fw_running;
+	u16 reserved;
```

Comment:
there's already reserved1 and reserved2 in the data structure, you may want to follow this convention and change accordingly.